RELEASE_NOTES.md

```diff
@@ -56,7 +56,23 @@ You can read more about this change on PR [#xxxx](https://github.com/decidim/dec
 
 These are one time actions that need to be done after the code is updated in the production database.
 
-### 3.1. [[TITLE OF THE ACTION]]
+### 3.1. Change of Valuator for Evaluator
+
+We have updated the terminology of Valuator at a code base level throughout the platform. The role of Valuator is now Evaluator. With this change also affects strings, i18n translations and so on.
+
+Implementors must run the following 3 tasks:
+
+```bash
+./bin/rails decidim:upgrade:decidim_update_valuators.rake
+./bin/rails decidim:upgrade:decidim_action_log_valuation_assignment.rake
+./bin/rails decidim:upgrade:decidim_paper_trail_valuation_assignment.rake
```

Comment:
Didn't realized before, but we need to drop the .rake at the end for this to work

```suggestion
./bin/rails decidim:upgrade:decidim_update_valuators
./bin/rails decidim:upgrade:decidim_action_log_valuation_assignment
./bin/rails decidim:upgrade:decidim_paper_trail_valuation_assignment
```